README.md

```diff
@@ -18,8 +18,8 @@ This repository enables building resin.io for various devices.
 
```

Comment:
README.md is modified in the first commit of this PR.
Guess this is a mistake to have it modified in this commit too?